ever2simple/converter.py

```diff
@@ -100,4 +108,19 @@ def _convert_json(self, notes):
         if self.simple_filename is None:
             sys.stdout.write(json.dumps(notes))
         else:
-            json.dump(notes, self.simple_filename)
+            with open(self.simple_filename, 'w') as output_file:
+                json.dump(notes, output_file)
+
+    def _convert_dir(self, notes):
+        if self.simple_filename is None:
+            sys.stdout.write(json.dumps(notes))
+        else:
+            if os.path.exists(self.simple_filename) and not os.path.isdir(self.simple_filename):
+                print '"%s" exists but is not a directory. %s' % self.simple_filename
+                sys.exit(1)
+            elif not os.path.exists(self.simple_filename):
+                os.makedirs(self.simple_filename)
+            for i, note in enumerate(notes):
+                output_file_path = os.path.join(self.simple_filename, str(i) + '.txt')
```

Comment:
That does seem like a lot of work for little gain. I'm cool with leaving them as numbers.